library/rsa.c

```diff
@@ -550,6 +550,11 @@ int mbedtls_rsa_gen_key(mbedtls_rsa_context *ctx,
         goto cleanup;
```

Comment:
Maybe remove `nbits < 128` here, and replace it by a compile-time check that `MBEDTLS_RSA_MIN_KEY_SIZE >= 128`?

Reply:
Done.